src/idl_gen_cpp.cpp

```diff
@@ -2972,15 +2974,34 @@ class CppGenerator : public BaseGenerator {
 
     cur_name_space_ = ns;
   }
-
-  const TypedFloatConstantGenerator float_const_gen_;
 };
 
 }  // namespace cpp
 
 bool GenerateCPP(const Parser &parser, const std::string &path,
                  const std::string &file_name) {
-  cpp::CppGenerator generator(parser, path, file_name);
+  cpp::IDLOptionsCpp opts(parser.opts);
+  // The '--cpp_std' argument could be extended (like ASAN):
+  // Example: "flatc --cpp_std c++17:option1:option2".
+  if (opts.cpp_std.empty() || opts.cpp_std == "c++") {
+    // Use a `standard` code generator.
```

Comment:
document what `g_cpp_std` is set to in this case?

Reply:
Fixed, g_cpp_std will be set to 11 (`c++11` is default value).